src/components/ItemsPerPage/ItemsSelect.tsx

```diff
@@ -0,0 +1,14 @@
+import { ItemsSelectProps } from './ItemsSelectTypes';
+
+export default function ItemsSelect({ getValue }: ItemsSelectProps) {
+  return (
+    <div className="per-page">
+      <p>Items per page:</p>
+      <select defaultValue={10} onChange={getValue}>
+        <option value="5">5</option>
+        <option value="10">10</option>
+        <option value="15">15</option>
```

Comment:
You can keep values in one variable, like `PAGINATION_FILTERS = [5, 10, 15]` or something similar